Bio/PDB/PDBParser.py

```diff
@@ -31,8 +31,7 @@
 class PDBParser(object):
     """Parse a PDB file and return a Structure object."""
 
-    def __init__(self, PERMISSIVE=True, get_header=False,
-                 structure_builder=None, QUIET=False):
+    def __init__(self, PERMISSIVE=True, structure_builder=None, QUIET=False):
```

Comment:
That looks like an API change (removing the ``get_header`` option) which is not backwards compatible.